### PR TITLE
Add spellchecker

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description
Introducing an automatic spellchecker.
Docusaurus already is scanning for broken links so we're already fine in this regard.

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+stateful
+webhooks
+webhook
+websocket
+ - blog/2024-09-13-first-documenation-iteration.md
+Fishjam
```
Comment (mironiasty) [inline .spelling:11]: Should this be in global override?

**Review by mironiasty — APPROVED** [review]

File: .spelling [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+params
+NodeJS
+e.g.
+js-server-sdk
+ - README.md
+Docusaurus
+ - guide/react-native/connecting.mdx
+pre-configured
+ - guide/react-native/installation.mdx
+TabItem
```
Comment (PiotrWodecki) [inline .spelling:26]: **suggestion (non-blocking):** We could move all of those to global scope. It would make writing new docs less tedious.

**Review by PiotrWodecki — APPROVED** [review]

File: .spelling (hunk 2 above)
Comment (czerwiukk) [reply .spelling:26]: i didn't notice I added it locally at first, moved everything to global scope

File: .spelling (hunk 1 above)
Comment (czerwiukk) [reply .spelling:11]: i moved everything to global scope as the words will probably repeat in different scopes

**Pull request merged**